src/game-coordinator/services/player-behavior-handler.service.spec.ts

```diff
@@ -0,0 +1,120 @@
+import { Events } from '@/events/events';
+import { Game, GameDocument, gameSchema } from '@/games/models/game';
+import { GamesService } from '@/games/services/games.service';
+import { PlayerSubstitutionService } from '@/games/services/player-substitution.service';
+import { Player, PlayerDocument, playerSchema } from '@/players/models/player';
+import { PlayersService } from '@/players/services/players.service';
+import { mongooseTestingModule } from '@/utils/testing-mongoose-module';
+import { getConnectionToken, MongooseModule } from '@nestjs/mongoose';
+import { Test, TestingModule } from '@nestjs/testing';
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import { Connection } from 'mongoose';
+import { PlayerBehaviorHandlerService } from './player-behavior-handler.service';
+// eslint-disable-next-line jest/no-mocks-import
+import { PlayersService as MockedPlayersService } from '@/players/services/__mocks__/players.service';
+// eslint-disable-next-line jest/no-mocks-import
```

Comment:
es-lolnt

Reply:
Yeah, I need to import mocked classes to have methods like `_crreateOne()` and `_reset()`.